test/receiver-node/package.json

```diff
@@ -7,9 +7,10 @@
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "author": "Josh Gavant",
-  "license": "Apache-2.0",
+  "license": "MIT",
```

Comment:
Can you keep with Apache? Is there a reason you suggest MIT?

Reply:
All our sdks are shipped with MIT, then why should the demo app be shipped with Apache? Is there any specific reason to do so?

Reply:
I like Apache better cause its more permissive. If we change to MIT we need to change the LICENSE file too.